besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2926,6 +2932,7 @@ private Runner synchronize(
             .storageProvider(keyValueStorageProvider(keyValueStorageName))
             .rpcEndpointService(rpcEndpointServiceImpl)
             .rpcMaxLogsRange(rpcMaxLogsRange)
+            .rpcGasCap(rpcGasCap)
```

Comment:
Wonder if this kind of thing should live in a configuration object. Not sure which one. Feels like rpcMaxLogsRange should live in the same one...probably an issue for another PR.